popgen/GvcfZipper.go

```diff
@@ -0,0 +1,80 @@
+package popgen
+
+import (
+	"sort"
+	"strings"
+	"log"
+	"github.com/vertgenlab/gonomics/vcf"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/dna"
+)
+
+//TajimaZipperVcf returns the Tajima's D value for each entry in a sorted bed slice using a pre-SORTED gVCF file for variant information.
+func TajimaZipperVcf(b []*bed.Bed, gVCFFile string) []float64 {
```

Comment:
I didn't think of this yesterday, but I think this will break if there are overlapping beds. Is that a dealbreaker? You may want to add a note in the documentation too.